### DISCUSSION TITLE
Is it possible to use OPA as a golang lib and also start its REST API server?

### DISCUSSION THREAD

**Discussion by challarao** [open-policy-agent/community#180] [Community]
Is the REST API module separate from the golang lib of OPA?

My use case is to have REST API work without repetition on my end but still use the direct lib capabilities for advanced stuff.

**Comment by anderseknert** [comment]
Yeah, the HTTP server isn't part of the Go API. If you'd want to run that from your code you'd probably need to mimic the way OPA manages that. I'm curious to hear more about your use case!
